gce_deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+export GITHUB_SHA=`git rev-parse HEAD`
+
+docker build -t gcr.io/$GCE_PROJECT/$GCE_INSTANCE-image:$GITHUB_SHA \
+          --build-arg GITHUB_SHA="$GITHUB_SHA" \
+          --build-arg GITHUB_REF="$GITHUB_REF" \
+          --build-arg DATABASE_HOSTNAME \
```

Comment:
@mfekadu You should be able to deploy using this script after using the new secrets file from the nimbus config secrets repo.